corehq/apps/app_manager/templates/app_manager/form_view_base.html

```diff
@@ -103,9 +103,18 @@
             var isModuleFilterEnabled = {{ is_module_filter_enabled|BOOL }};
             var formFilter = {{ form.form_filter|JSON }};
             var allOtherFormsRequireACase = {{ form.all_other_forms_require_a_case|BOOL }};
+            var formFilterReferencesCase = {{ form_filter_references_case|BOOL }};
             var formFilterAllowed = ko.computed(function () {
```

Comment:
Hey,
Is this changing the behavior of the UI to allow you to enter form filter conditions when previously you weren't? And potential bad configurations we disallowed before are now allowed, but caught on build?

It's possible that this should be lumped into the module display condition feature preview instead of allowing more configurations productized.

Longer term, the way I see this productized is always allowing form filter conditions but using special syntaxing to only autocomplete to references that are available